config/crd/bases/enterprise.splunk.com_clustermasters.yaml

```diff
@@ -990,6 +990,43 @@ spec:
                       type: object
                     type: array
                 type: object
+              clusterManagerRef:
```

Comment:
just want to understand why is `clusterManagerRef` here in the clustermasters.yaml file? CC: @vivekr-splunk / @sgontla

Reply:
This is because`clusterManagerRef` is part of `common_types`, so all CRDs have these Refs. This is similar to `clusterMasterRef` :

Here's the snippet from `common_types.go`:
```
type CommonSplunkSpec struct { 

LicenseMasterRef corev1.ObjectReference `json:"licenseMasterRef,omitempty"`
LicenseManagerRef corev1.ObjectReference `json:"licenseManagerRef,omitempty"`
ClusterMasterRef corev1.ObjectReference `json:"clusterMasterRef"`
ClusterManagerRef corev1.ObjectReference `json:"clusterManagerRef"`
MonitoringConsoleRef corev1.ObjectReference `json:"monitoringConsoleRef"`

	... etc ...
```